flask_appbuilder/api/convert.py

```diff
@@ -1,46 +1,45 @@
-from typing import List, Optional, Type
+from typing import Any, Callable, Dict, List, Optional, Type
 
 from flask_appbuilder.models.sqla import Model
 from flask_appbuilder.models.sqla.interface import SQLAInterface
-from marshmallow import fields
+from marshmallow import fields, Schema
 from marshmallow.fields import Field
-from marshmallow_enum import EnumField
 from marshmallow_sqlalchemy import field_for
 from marshmallow_sqlalchemy import SQLAlchemyAutoSchema
 
 
 class TreeNode:
-    def __init__(self, data):
-        self.data = data
-        self.childs = list()
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.childs: List["TreeNode"] = []
```

Comment:
nit: at the risk of introducing a breaking change, maybe this should be `self.children`